lib/desktop/quicksettings/quick_settings.dart

```diff
@@ -342,56 +341,58 @@ class QuickSettingsState extends State<QuickSettings> {
             buildTile(Icons.airplanemode_inactive, LocaleStrings.pangolin.qsAirplanemode, changeColor),
             buildTile(Icons.invert_colors_off, LocaleStrings.pangolin.qsInvertcolors, changeColor),
             buildTile(Icons.language, LocaleStrings.pangolin.qsChangelanguage, () {
-              switch (Localizations.localeOf(context).toString()) {
-                case "en_US":
+              // String sVar = Localizations.localeOf(context).toString();
+              String sVar = Pangolin.locale.countryCode;
```

Comment:
same thing as before, please dont use such algorithms as they are really hard to maintain

Reply:
removed smells code. Yet with switch case 😕